--- libetonyek.spec.fail	2023-08-24 09:25:59.000000000 +0200
+++ libetonyek.spec.success	2023-08-24 09:36:08.000000000 +0200
@@ -2,7 +2,7 @@
 ## (rpmautospec version 0.3.5)
 ## RPMAUTOSPEC: autorelease, autochangelog
 %define autorelease(e:s:pb:n) %{?-p:0.}%{lua:
-    release_number = 2;
+    release_number = 3;
     base_release_number = tonumber(rpm.expand("%{?-b*}%{!?-b:1}"));
     print(release_number + base_release_number - 1);
 }%{?-e:.%{-e*}}%{?-s:.%{-s*}}%{!?-n:%{?dist}}
@@ -72,7 +72,7 @@
 
 %build
 autoreconf -i -f
-%configure --disable-silent-rules --disable-static --disable-werror --with-mdds=2.0
+%configure --disable-silent-rules --disable-static --disable-werror --with-mdds=2.1
 sed -i \
     -e 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' \
     -e 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' \
@@ -137,6 +137,9 @@
 %{_mandir}/man1/pages2text.1*
 
 %changelog
+* Thu Aug 24 2023 Mattia Verga <mattia.verga@proton.me> - 0.1.11~20230802.git9c3a8cb-3
+- Change mdds req also in configure parameters
+
 * Thu Aug 24 2023 Mattia Verga <mattia.verga@proton.me> - 0.1.11~20230802.git9c3a8cb-2
 - Fix mdds version requirement
 
